src/helpers.js

```diff
@@ -178,8 +178,8 @@ const helpers = {
    * @inner
    */
   getServerURL() {
-    const server = storage.getItem('wallet:server');
-    const defaultServer = storage.getItem('wallet:defaultServer');
+    const server = StorageProxy.getStorage().getItem('wallet:server');
```

Comment:
I think it is best to create a `const wallet = StorageProxy.getStorage()` to avoid duplicate code.

Reply:
Done